test/libsolidity/syntaxTests/inlineAssembly/evm_constantinople.sol

```diff
@@ -0,0 +1,22 @@
+contract C {
+    function f() view external returns (uint ret) {
+        assembly {
+            ret := shl(gas(), 5)
+            ret := shr(ret, 2)
+            ret := sar(ret, 2)
+        }
+    }
+    function g() external returns (address ret) {
+        assembly {
+            ret := create2(0, 0, 0, 0)
+        }
+    }
+    function h() pure external returns (uint ret) {
+        assembly {
+            ret := chainid()
```

Comment:
This one is istanbul isn't it? That'd go along with the test failure we're getting.

Reply:
Yes, but I already pushed a fix but somehow it is not showing up :(

Reply:
Weird - for me on a local git checkout the file is also still the same, no matter how much I pull on it...